lib/arproxy/config.rb

```diff
@@ -14,12 +16,23 @@ def initialize
     end
 
     def use(proxy_class, *options)
+      if proxy_class.is_a?(Class) && proxy_class.ancestors.include?(Arproxy::Base)
+        # TODO: Add a migration guide URL
```

Comment:
I will add an url like following into the error message after merging

https://github.com/cookpad/arproxy/blob/62e7e9e8ea2f68963e2f0f6dfaf5986a6f10d052/README.md#upgrading-guide-from-v0x-to-v1

Reply:
Good idea. Added UPGRADING.md 31d8af176ef69482fd25bc0dd58277ddcf5232f8